app/pages/lab/project.cjsx

```diff
@@ -54,11 +54,15 @@ EditProjectPage = React.createClass
             <PromiseRenderer promise={@props.project.get 'workflows'}>{(workflows) =>
               <ul className="nav-list">
                 {for workflow in workflows
-                  workflowLinkParams = Object.create linkParams
-                  workflowLinkParams.workflowID = workflow.id
-                  <li key={workflow.id}>
-                    <Link to="edit-project-workflow" params={workflowLinkParams} className="nav-list-item">{workflow.display_name}</Link>
-                  </li>}
+                  renderWorkflowListItem = ->
```

Comment:
Looks like if you have multiple workflows defined it repeatedly renders a link to the the first one. Pulling the function expression out of the loop and making it pure fixed it for me (same for the subject set link function)